es51x/src/main/scala/tech/beshu/ror/es/request/SearchQueryDecorator.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.es.request
+
+import org.apache.logging.log4j.scala.Logging
+import org.elasticsearch.action.search.SearchRequest
+import org.elasticsearch.index.query.{QueryBuilder, QueryBuilders}
+import tech.beshu.ror.Constants
+import tech.beshu.ror.accesscontrol.domain.Filter
+
+object SearchQueryDecorator extends Logging {
```

Comment:
this name suggest that this is decorator pattern, but it's not, so it could be misleading. Mayber we can rename it (and refactor a bit) to SearchRequestOps class?